.github/workflows/build_and_test.yml

```diff
@@ -18,8 +18,8 @@ env:
 
 jobs:
   tag:
-    runs-on: dev
-    container: 369495373322.dkr.ecr.eu-central-1.amazonaws.com/base:latest
+    runs-on: [ dev, x64 ]
```

Comment:
Added the self-hosted tag, will leave out the OS as that might cause confusion in combination with the container runtime

Reply:
I mean it would be nice to be explicit about not running the Github default OS though. If dropping a container, it rather might come as a surprise that these runners don't have the same tools around as the default ones.